util/annotations/annotations.go

```diff
@@ -43,20 +42,36 @@ func (a *Annotations) Add(err error) Annotations {
 	if *a == nil {
 		*a = Annotations{}
 	}
+	prevErr, exists := (*a)[err.Error()]
+	if exists {
```

Comment:
This could use the short form, couldn't it?

```suggestion
	if prevErr, exists := (*a)[err.Error()]; exists {
```